wrappers/here.rb

```diff
@@ -455,13 +455,24 @@ def here_strict_restriction(param_value)
     end
 
     ##
-    # < 1000km: 15, 1500km: 10, 2000km : 5
-    def max_srcs(dist_km)
+    # motorway or toll -> dist_km < 1000km: 15, 1500km: 10, 2000km: 5
+    # !motorway or !toll -> dist_km < 1000km: 15, 1000km: 6, 2000km: 3, 3000km: 1
+    def max_srcs(dist_km, options)
       if dist_km <= 1_000
         15
+      elsif options[:motorway] == false || options[:toll] == false
+        [8 - (dist_km * 0.0025).floor, 1].max
       else
         [25 - (dist_km / 100).floor, 1].max
       end
     end
+
+    def max_dsts(dist_km, dsts, options)
```

Comment:
```suggestion
    def max_dsts(dist_km, dsts, options = {})
```

Reply:
I thought about it as it should be more simple to call in tests, but I wanted to force the call with options as in real case there is always options and that splitting calculations is based on it. @fab-girard any preference?